server/migrations/versions/448548fd949a_.py

```diff
@@ -0,0 +1,36 @@
+"""empty message
```

Comment:
@sina-jamshidi @bugatti12345 
Should we include this file in git? It seems it changes every time we migrate.

Reply:
I think we should be including migrations because the idea is that all team members should have exactly the same database